Meetings/2022-05-03.md

```diff
@@ -0,0 +1,23 @@
+# TSC Meeting 2022 May 3rd @12PM PDT / 12PM MST / 3PM EDT
```

Comment:
Beware Daylight Savings Time. I think for this group we're sticking with 12PM Pacific as the standing time so AZ folks will shift over, but everyone else stays the same.